[2024-12-16T20:26:42.511+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-16T20:26:42.524+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_FOR_AWS.second_task scheduled__2024-12-15T09:00:00+00:00 [queued]>
[2024-12-16T20:26:42.529+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_FOR_AWS.second_task scheduled__2024-12-15T09:00:00+00:00 [queued]>
[2024-12-16T20:26:42.530+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-12-16T20:26:42.539+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): second_task> on 2024-12-15 09:00:00+00:00
[2024-12-16T20:26:42.544+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=67) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-16T20:26:42.546+0000] {standard_task_runner.py:72} INFO - Started process 71 to run task
[2024-12-16T20:26:42.547+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_FOR_AWS', 'second_task', 'scheduled__2024-12-15T09:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/MI_PRIMER_DAG.py', '--cfg-path', '/tmp/tmpszn7s2kq']
[2024-12-16T20:26:42.547+0000] {standard_task_runner.py:105} INFO - Job 152: Subtask second_task
[2024-12-16T20:26:42.586+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_FOR_AWS.second_task scheduled__2024-12-15T09:00:00+00:00 [running]> on host 590f5de10d76
[2024-12-16T20:26:42.640+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_FOR_AWS' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T09:00:00+00:00'
[2024-12-16T20:26:42.641+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-16T20:26:42.641+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-16T20:26:42.641+0000] {logging_mixin.py:190} INFO - Current task name:second_task state:running start_date:2024-12-16 20:26:42.524685+00:00
[2024-12-16T20:26:42.642+0000] {logging_mixin.py:190} INFO - Dag name:DAG_FOR_AWS and current dag run status:running
[2024-12-16T20:26:42.642+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-16T20:26:42.643+0000] {logging_mixin.py:190} INFO - Second_task
[2024-12-16T20:26:42.643+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-16T20:26:42.649+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-16T20:26:42.649+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_FOR_AWS, task_id=second_task, run_id=scheduled__2024-12-15T09:00:00+00:00, execution_date=20241215T090000, start_date=20241216T202642, end_date=20241216T202642
[2024-12-16T20:26:42.656+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-16T20:26:42.656+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-16T20:26:42.656+0000] {logging_mixin.py:190} INFO - Dag name:DAG_FOR_AWS queued_at:2024-12-16 20:26:40.373099+00:00
[2024-12-16T20:26:42.657+0000] {logging_mixin.py:190} INFO - Task hostname:590f5de10d76 operator:PythonOperator
[2024-12-16T20:26:42.680+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-16T20:26:42.697+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-16T20:26:42.700+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
